src/components/Banner/BoosterBanner/Buttons.style.tsx

```diff
@@ -0,0 +1,42 @@
+import styled, { css } from 'styled-components';
+import {
+  LargeFilledButton,
+  LargeOutlinedButton,
+} from 'components/ButtonSystem';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+import { ButtonType } from 'assets/theme/buttons';
+
+const BaseButtonStyles = css`
+  min-width: 11rem;
+  margin: 0 0.5rem 0 0.5rem;
+  @media (max-width: ${materialSMBreakpoint}) {
+    font-size: 0.9rem;
+    padding: 0.75rem;
+    height: 2.5rem;
+    width: 100%;
+    max-width: 16rem;
```

Comment:
Not sure whether the button width should be 100% as soon as the screen size falls below 600. It just looks really wide so I implement max-width.